java/src/main/java/io/vertx/ext/eventbus/client/options/PemTrustOptions.java

```diff
@@ -0,0 +1,36 @@
+package io.vertx.ext.eventbus.client.options;
+
+import javax.xml.bind.DatatypeConverter;
+import java.io.ByteArrayInputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateFactory;
+
+/**
+ * @author <a href="mailto:pl@linux.com">Phil Lehmann</a>
+ */
+public class PemTrustOptions extends TrustOptions {
```

Comment:
so for PemTrust I think that netty provide an implementation (I didn't know it when I implemented it in Vert.x)

Reply:
What I found was `PemPrivateKey` which was added for netty servers (https://github.com/netty/netty/pull/5377). I could find no class for PEM certs, which is pretty similar though.